terraform/github-to-gitlab-mirror/main.tf

```diff
@@ -0,0 +1,59 @@
+locals {
+  user = "nix-community"
+  blacklist = [
+    # invalid name
+    ".github",
+    # do already exists
+    "emacs-overlay",
+    "docker-nixpkgs",
+    "nixos-generators",
+    "poetry2nix",
+  ]
+  # remove blacklist
+  filteredRepos = [
+    for repo in data.github_repositories.repos.full_names : repo if !contains(local.blacklist, element(split("/", repo), 1))
+  ]
+}
+
+data "github_repositories" "repos" {
+  query = "user:${local.user} is:public"
+}
+
+data "github_repositories" "non-archived-repos" {
+  query = "user:${local.user} is:public archived:false"
+}
+
+resource "gitlab_group" "group" {
+  name             = local.user
+  path             = local.user
+  visibility_level = "public"
+}
+
+resource "gitlab_project" "repos" {
```

Comment:
Can we set `issues_enabled = false` and `merge_requests_enabled = false` on these repos?